mlos_core/mlos_core/optimizers/optimizer.py

```diff
@@ -26,7 +26,7 @@ class BaseOptimizer(metaclass=ABCMeta):
 
     def __init__(self, *,
                  parameter_space: ConfigSpace.ConfigurationSpace,
-                 optimization_targets: List[str],
+                 optimization_targets: Optional[Union[str, List[str]]] = None,
```

Comment:
Why is this taking `None`?

Reply:
Makes sense. But that's a separate change. Please submit a different PR for that so we can focus tests on that. Thanks!